docs/vertx-runtime/master.adoc

```diff
@@ -255,11 +255,21 @@ include::topics/secured-mission-resources.adoc[leveloffset=+3]
 //Minimum Viable App
 //Add-ons
 //Building your app
+// TODO: make content properly reusable
+//creating an app from scratch
```

Comment:
Does the TODO still make sense? :-)

Reply:
@Ladicek Oh, right. That one is no longer necessary. Guess I should remove that :)